test/triplex/triplex_test.exs

```diff
@@ -154,6 +158,21 @@ defmodule TriplexTest do
     end
   end
 
+  @tag :new
+  test "migrate/2 works when invoked with list of tenants from all/1" do
+    for repo <- @repos do
+      Triplex.create("lala", repo)
+      Triplex.create("lili", repo)
+      Triplex.create("lolo", repo)
+
+      result =
+        Triplex.all(repo)
+        |> Enum.each(fn t -> Triplex.migrate(t) end)
+
+      assert result == :ok
```

Comment:
If I'm not mistaken, `Enum.each` always returns `:ok`, so the test is not actually testing if the migrations ran... I guess that you would need a `Enum.map` for that, and test if all 3 tenants got migrated

Reply:
Again ... not sure what I was thinking .. I have changed it.